lib/jobs/create-default-pollers.js

```diff
@@ -44,41 +44,63 @@ function createDefaultPollersJobFactory(
     }
 
     util.inherits(CreateDefaultPollersJob, BaseJob);
-    
+
+    /**
+     * @memberOf CreateDefaultPollersJob
+     * @param {String} nodeId: node id string
+     * @param {Object} poller: poller to be created
+     * @param {String} obmName: OBM service name required to create pollers
+     */
+    CreateDefaultPollersJob.prototype._createWorkitem = function(nodeId, poller, obmName) {
+        assert.isMongoId(nodeId, 'nodeId');
```

Comment:
It's better to remove this check, as nodeId may not always be MongoID since RackHD supports multiple kind of database.

Reply:
@yyscamper  Good catch